controllers/csr_controller.go

```diff
@@ -141,7 +150,7 @@ func (r *CSRReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.R
 		return ctrl.Result{Requeue: true, RequeueAfter: time.Minute}, errIssuerNotReady
 	}
 
-	l.Info("Signing", "csr", req)
+	l.Info("Signing")
```

Comment:
Why we don't pint what we sign anymore? I think it is valuable info in debugging.

Reply:
I agree and reverted this change.